ocaml/xcp-rrdd/bin/rrdd/xcp_rrdd.ml

```diff
@@ -545,6 +557,34 @@ let bytes_per_mem_vm = 1024
 
 let mem_vm_writer_pages = ((max_supported_vms * bytes_per_mem_vm) + 4095) / 4096
 
+let res_error fmt = Printf.kprintf (fun msg -> Result.error (`Msg msg)) fmt
```

Comment:
The `Msg` part is not needed - it was in my code because I just copied it from somewhere else. The error is a simple string.

Reply:
Fixed